tracee-rules/signatures/rego/regosig/traceerego.go

```diff
@@ -31,10 +31,10 @@ const querySelectedEvents string = "data.main.tracee_selected_events"
 const queryMetadata string = "data.main.__rego_metadoc__"
 
 // NewRegoSignature creates a new RegoSignature with the provided rego code string
-func NewRegoSignature(regoCode string) (types.Signature, error) {
+func NewRegoSignature(regoCode string, regoHelpersCode string) (types.Signature, error) {
 	var err error
 	res := RegoSignature{}
-	res.compiledRego, err = ast.CompileModules(map[string]string{"sig": regoCode})
+	res.compiledRego, err = ast.CompileModules(map[string]string{"sig": regoCode, "helpers": regoHelpersCode})
```

Comment:
helpers should be optional. please state that in the function comment/docstring and check if helpers was provided before loading it